sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -0,0 +1,87 @@
+package azidentity
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/mock"
+)
+
+const (
+	certificatePath      = "../../../testPEM.pem"
```

Comment:
NOTE: I am working on adding a local PEM file for testing and will add it shortly